src/main/java/com/elixr/poc/service/UserService.java

```diff
@@ -0,0 +1,41 @@
+package com.elixr.poc.service;
+
+import com.elixr.poc.constants.ApplicationConstants;
+import com.elixr.poc.repository.UserRepository;
+import com.elixr.poc.rest.response.DeleteResponse;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+/*
+Service layer
+ */
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+    public int REQUEST_VALUE;
+
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    /*
+    Deleting the user by the userId.
+     */
+    public DeleteResponse deleteUserDetails(UUID userId) {
+        DeleteResponse deleteResponse = new DeleteResponse();
+        boolean userRecordExists = userRepository.existsById(userId);
+        if (userRecordExists) {
+            userRepository.deleteById(userId);
+            REQUEST_VALUE = 200;
```

Comment:
You are not using this variable

Reply:
Yes I'm using it, to show the status as (200 OK) in postman once the deletion is done.
In line 35, I'm using to show the status as (404 NOT FOUND) in postman